src/hooks/useCountries.jsx

```diff
@@ -1,16 +1,20 @@
 'use client'
 
 import { useQuery} from '@tanstack/react-query'
-import fetchData from './fetchData'
+import fetchData from '../app/api/fetchData'
 
 const useCountries = (options) => {
   return useQuery({
     queryKey: ['countries'],
-    queryFn: async () => {
+    queryFn: () => {
       return fetchData('/all', options)
     },
     ...options
   })
 }
 
+export const getUseQueryKey = () => {
```

Comment:
```suggestion
export const getUseCountriesQueryKey = () => {
```
We wcześniejszym komentarzu zasugerowałem taką nazwę. Pomyliłem się przepraszam, chodziło mi o `getUseCountriesQueryKey`